functions.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+function getGuitarList()
```

Comment:
missing return type. the return of this function should `PDO`

Reply:
and looking at the function name, it doesnt make sense any more, this function doesnt get a list of guitars, it gets the database connection